cmd/access-control/access-control.go

```diff
@@ -0,0 +1,234 @@
+package accesscontrol
```

Comment:
imo, this should be inside `cmd/catalyst-node`. the (undocumented) project structure so far has been that `cmd/<package>` should be able to run as `go run <package>` without any dependencies. since access-control is adding another handler to catalyst-node server.

wdyt? i really liked the restructuring in another PR: #163

Reply:
Yes, I tried that but had to use `go build -o ./bin/livepeer-catalyst-node cmd/catalyst-node/catalyst-node.go cmd/catalyst-node/access-control.go` and was not sure where else to add this.

> wdyt? i really liked the restructuring in another PR: https://github.com/livepeer/catalyst/pull/163
Yeah me too! I was outpaced by additions to `catalyst-node.go`. I plan to land the feature here then proceed, as a follow-up to refactor the handlers.

Reply:
> Yes, I tried that but had to use `go build -o ./bin/livepeer-catalyst-node cmd/catalyst-node/catalyst-node.go cmd/catalyst-node/access-control.go` and was not sure where else to add this.

was this true for #163 as well? it might have to do with naming as `cmd/catalyst-node/access-control.go` instead of `cmd/catalyst-node/handlers/access-control.go` maybe?

Reply:
On #163, IIRC the handlers were as well extracted to the `handlers` package.

Reply:
Getting this, if everything is inside `cmd/catalyst-node/access-control.go` under the `main` package
![image](https://user-images.githubusercontent.com/231488/194827042-7e89fe7c-9e8a-496c-a58a-7ad0f7848456.png)

Reply:
pushed in a fix at [57f3743](https://github.com/livepeer/catalyst/pull/188/commits/57f3743a6566c27481e73a5bfa33b3f1fe616121)